.circleci/config.yml

```diff
@@ -267,21 +267,21 @@ workflows:
           filters:
             branches:
               only:
-                - /^release\/.*/
+                - /^(release|feature)\/.*/
```

Comment:
```suggestion
                - /^release\/.*/
```

Reply:
I just added this to make sure dependency updates have all e2e tests working as expected.

Reply:
I think we can use a different prefix for dependency update task e.g `dep-update`?

Reply:
Agree with dep-update. Also add `renovate/` here if we are doing it in the future?